src/Faction/formulas/donation.ts

```diff
@@ -1,7 +1,31 @@
+import { Person } from "@nsdefs";
+
 import { CONSTANTS } from "../../Constants";
 import { currentNodeMults } from "../../BitNode/BitNodeMultipliers";
-import { Person as IPerson } from "@nsdefs";
```

Comment:
The reason Person was named differently here is because there are two separate `Person` types.

`Person` from `@nsdefs` is an interface which just defines a minimum personlike structure for players passing in person objects to formulas functions.

`Person` is also the name of an actual game class, which does fulfill the Person interface but also has member functions and some other properties that aren't visible to players through the API.

So typically when we use the ns API versions of Person/Player/Sleeve, we don't refer to them using those names directly because we want the code to be clear that we're not talking about the ingame class, we're talking about the ns API interface version. The exact name doesn't really matter (could call it NSPerson, I have been fine keeping these specific instances as IPerson as well since the I is actually conveying information in this case)

Reply:
Thanks for the explanation :)

It looks a bit strange that the class and interface have the same name which forces us to use an alias for the interface when importing it in-game.
Maybe naming the class as something like `AbstractPerson` would help both in not having to import as `IPerson`  the interface and also for code readability (when you see `AbstractPerson` you know, that this is a class with some basic implementation/attributes representing either a player or a sleeve)
wdyt?

Reply:
I'm pretty OK with the way things are. There isn't a better name than Person, or Script, or Server, etc. At the same time, there's no reason to name the public interface that we derive from these class anything else, either. It only causes friction in the (few) places where both are used.